src/processes/scoring/formula/initial-score-value.js

```diff
@@ -60,6 +60,7 @@ const initDataUserScore = (userId, timestamp) => ({
   last_daily_process: '',
   created_at: timestamp,
   updated_at: timestamp,
+  w_pu_user: 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `w_pu_user: 1` seems to be a new field in the user score initialization. However, it's not clear what this field represents based on the provided context. Please ensure that this change is well-documented and communicated to other developers who might interact with this code or the data it produces. If this field has any constraints or special behaviors associated with it, those should also be documented.


<!-- This is an auto-generated comment by OSS CodeRabbit -->